src/utils/src/arg_parser.rs

```diff
@@ -208,9 +208,7 @@ impl<'a> Arguments<'a> {
     fn value_of(&self, arg_name: &'static str) -> Option<&Value> {
         if let Some(argument) = self.args.get(arg_name) {
             match &argument.user_value {
-                Some(val) => {
-                    return Some(val);
-                }
+                Some(val) => return Some(val),
                 None => return argument.default_value.as_ref(),
```

Comment:
I'm biased towards one-liners and chained operators, so dropping in a Rust style nit:
```suggestion
       self.args.get(arg_name).and_then(|argument| {
            argument
                .user_value
                .as_ref()
                .or_else(|| argument.default_value.as_ref())
        })
```

Reply:
@aghecenco changes done 👍 , also changed some other functions to match this style 🙂.